src/main/scala/cromwell/engine/WorkflowDescriptor.scala

```diff
@@ -61,16 +62,12 @@ case class WorkflowDescriptor(id: WorkflowId,
   lazy val fileHasher: FileHasher = { wdlFile: WdlFile => SymbolHash(ioManager.hash(wdlFile.value)) }
 
   // GCS FS with the workflow working directory as root
-  val gcsFilesystem = for {
-    interface <- gcsInterface
-    fs <- Try(GcsFileSystem.instance(interface, wfContext.root))
-  } yield fs
+  lazy val gcsFilesystem = Try(GcsFileSystem.instance(gcsInterface, wfContext.root))
```

Comment:
I know I'm not saying this is perfect, it was a first step towards differentiating between `Backend`s and `FileSystem`s. We can definitely codify this better, however because we allow workflow options to be passed containing refresh tokens and so on, each workflow potentially has different ways enabled to communicate with different FileSystems, which is why this ended up in the WorkflowDescriptor.